src/icons/emojis/anxiousEmoji.ts

```diff
@@ -0,0 +1,4 @@
+import { AnxiousEmoji } from './svgs';
+import { useIcon, IconProps } from '../../shared-components/icon';
+
+export default (props: IconProps) => useIcon(AnxiousEmoji, AnxiousEmoji, props);
```

Comment:
The default behavior of `create-index` is to generate an `index.ts` file comprising `default` exports renamed as their "safe" named export value. It automatically picks up that this should be `export { default as AnxiousEmoji }` from the filename. 

There's good React reasons to not use anonymous default exports--the component shows up as `Anonymous` in the Component tree in dev tools--but since the underlying components in `useIcon` will be named (by `@svgr/webpack`), this seemed a fine trade-off to make for maintaining compatibility with `create-index`. 
<img width="334" alt="Screen Shot 2020-11-18 at 10 21 49 AM" src="https://user-images.githubusercontent.com/13544620/99557137-f43e9d00-2987-11eb-954f-2eff30829345.png">

Moreover, anonymous default exports reduces auto-import name-spacing noise a bit. (Currently if you add `AnxiousEmoji` in a file, the auto-import will suggests both `../../src/icons` and `../../src/icons/emojis` as relative imports, and TypeScript definitions work best with relative imports.)